### PR TITLE
feat: add contact policy api double

### PR DESCRIPTION
**Pull request by ramniel**
Integrate contact-policy API (https://apis.fromdoppler.com/contact-policies/swagger/index.html) with double

### CODE REVIEW

File: src/services/doppler-contact-policy-api-client.test.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+describe('HttpDopplerContactPolicyApiClient', () => {
+  afterEach(cleanup);
+
+  //TODO: Implement real tests
```
Comment (andresmoschini) [inline src/services/doppler-contact-policy-api-client.test.ts:7]: Will it be done before or after the merge? 😛

**Comment by andresmoschini** [comment]
Looks fine, I would integrate this with the UI in this same PR, but if you prefer to split in this way, it is ok for me

File: src/services/doppler-contact-policy-api-client.test.ts (hunk 1 above)
Comment (ramniel) [reply src/services/doppler-contact-policy-api-client.test.ts:7]: After

Comment (andresmoschini) [reply src/services/doppler-contact-policy-api-client.test.ts:7]: 😕 Mmmm, in that case, I think that you could remove the file, it is totally unuseful at the moment.

Or..., you can add tests before the merge, you can mock Axios response and test the URLs requested, the mappers, how you deal with the errors, etc.

**Comment by jcamposmk** [comment]
:+1:

**Pull request merged**